addons/mrp/models/mrp_production.py

```diff
@@ -65,7 +65,9 @@ def _get_default_date_planned_finished(self):
     def _get_default_date_planned_start(self):
         if self.env.context.get('default_date_deadline'):
             return fields.Datetime.to_datetime(self.env.context.get('default_date_deadline'))
-        return datetime.datetime.now()
+        regex_to_round_up = re.compile("[0-9]+-[0-9]+-[0-9]+T[0-9]+")
```

Comment:
why not just use:
now = datetime.datetime.now().replace(minute=0, second=0, microsecond=0)
now = now + datetime.timedelta(hours=1) (you should also use add from odoo date_utils)
return now.to_string() -> Better it uses the format defined by the user preference